pkg/controller/controller_instance.go

```diff
@@ -1582,6 +959,7 @@ func setServiceInstanceConditionInternal(toUpdate *v1beta1.ServiceInstance,
 	t metav1.Time) {
 
 	pcb := pretty.NewContextBuilder(pretty.ServiceInstance, toUpdate.Namespace, toUpdate.Name)
+	glog.Info(pcb.Message(message))
 	glog.V(5).Info(pcb.Messagef(
```

Comment:
Looks like message should be added to this info line vs it's own line?

Reply:
Personally I like it as two lines, since it's two separate pieces of information: the information being logged vs the condition transitioning.

Not 100% sold on logging the message here, though; it keeps us from having to make an additional `pcb` and a log call in each of our `processX` methods, but feels a little hacky? Thoughts on if I made a `logAndRecordEvent()` function instead that had both the glog and the event recorder calls?

Reply:
The only reason we did not do that was the line logged for glog would be logAndRecordEvent which is not super helpful

Reply:
Oh yeah :sweat_smile:

Reply:
@n3wscott looks like we can use `glog.InfoDepth(1, "foobar")` to have the filename/line number be one layer up on the call stack.